src/cli/profile_loader.ts

```diff
@@ -12,7 +12,18 @@ export default class ProfileLoader {
   }
 
   async getDefinitions(): Promise<Dictionary<string>> {
-    const definitionsFilePath = path.join(this.directory, 'cucumber.js')
+    const isConfigPresent = process.argv.indexOf('--config') || process.argv.indexOf('-c')
+    let configFilePath = null
+    if (isConfigPresent > 0) {
+      configFilePath = path.join(
+        this.directory,
+        process.argv[isConfigPresent + 1].toString()
+      )
+    } else {
+      configFilePath = path.join(this.directory, 'cucumber.js')
+    }
+    const definitionsFilePath = configFilePath
```

Comment:
Let's remove this reassignment.

Reply:
i am noty able to push new branch .. i had 2 changes somehow i tried to edit the file and createpr .. but its nont allowing me to combine all changes into patch can you suggest

Reply:
and where do i update the documentation ? also @aslakhellesoy  ..i need 2 patches to go together .. 
Patch 2 and 3 i guess under my name .. one for arg  parser update one to provide the args and pass the file path

Reply:
You could start over from master:

```
git checkout -b config-option
# hack hack
git commit -m "bla bla"
# hack hack
git commit -m "bla bla"
# hack hack
git commit -m "bla bla"
git push

# Open the URL in your terminal and create a pull request

# hack hack
git commit -m "bla bla"
git push

# Now you just pushed a new commit to the branch of your pull request.
```

More details here: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request

Reply:
Remote: Permission to cucumber/cucumber-js.git denied to deepziem. unable to access 'https://github.com/cucumber/cucumber-js.git/': The requested URL returned error: 403

Reply:
@aslakhellesoy can you help here.. help update commits

Reply:
You have  to push to your [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) - looks like you've figured it out in #1435

Shall we close this one and leave #1435 open?

Reply:
both are required both are different files, not sure how to merge 2 patches or both need to merge to master.. i am not able to use desktop thats why ..

Reply:
Maybe take some time to learn how to use git and GitHub wold help?

Reply:
Hang on - are you editing files in the browser using the GitHub editor?

Reply:
yes .. i am .. i am not able to branch or push from desktop .. not sure why

Reply:
hi @aslakhellesoy .. can you approve the changes pls? is there any issue still

Reply:
Hi @deepziem

Can you please merge the changes from #1435 into this PR?

Would also like to add a feature test for this change.

As for deasync, please create a new issue so that can be discussed seperately